### DISCUSSION TITLE
Avoid over-positioning of selected chat reply

### DISCUSSION THREAD

**Discussion by Moshyfawn** [zed-industries/zed#23918] [Feature Requests]
If there's not enough space left to scroll to the bottom of the chat box, selecting a message reply scrolls it as far as it can and tries to position it at the top of the container, leaving the message in the middle of the chat history, which is hard to find. I believe that highlighting the selected message, as described in #10024, might be enough to both clean up this UX and avoid too much scrolling in general, helping the user to better maintain focus.

https://github.com/zed-industries/zed/assets/16290753/3c446229-068a-46dd-873c-9e46787f46dd

**Comment by maxdeviant** [comment]
We can evaluate this once we revisit Zed's chat functionality.
